client/src/components/Visualization/Visualization.js

```diff
@@ -0,0 +1,38 @@
+import './Visualization.css';
+
+import { Col, Container, Row } from 'react-grid-system';
+
+import BarChart from './charts/BarChart';
+import Interpolation from './charts/Interpolation';
+import React from 'react';
+import ZoomLine from './charts/ZoomLine';
+import tardigrade from '../../images/tardigrade.png';
+
+const Visualization = () => {
+  const chartMap = {
+    'Zoomable Line': <ZoomLine />,
+    Interpolation: <Interpolation />,
```

Comment:
Should this be a string for consistency? Don't think it matters b/c json but still.